recipes/racket/build.sh

```diff
@@ -0,0 +1,17 @@
+# Start with a minimal build without any packages.
+# Avoid building any packages since they could potentially
+# require a more recent version of the base language than
+# is in the current source directory.
+# Packages can be added once the package manager is configured
+# to work for this particular version number.
+make unix-style PREFIX=$PREFIX PKGS=""
```

Comment:
Is there a test suite we can run (possibly even a small portion of a test suite)?